style.css

```diff
@@ -373,3 +373,70 @@ section {
     margin-bottom: 250px;
   }
 }
+
+.contact-us {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  justify-content: center;
+  background: url('/images/contact-background-svg.svg');
+  background-size: 5vmax;
+  background-repeat: no-repeat;
+  background-position: center right;
+  background-position-y: 850px;
+}
+
+p.contact {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
```

Comment:
- [ ] I see you have the font-weight 700. But it does not look as bold as in Figma, right? 🤔 

Maybe you should check how are you importing the font. If you use google fonts for this, make sure you downloaded the font-weights you are using. 

![Captura de Pantalla 2022-09-27 a la(s) 8 40 40 a  m](https://user-images.githubusercontent.com/86022690/192545662-01421295-d609-4290-8941-2612d7cbc377.png)

In the link to the font, you should see the font weight. This is an example with Poppins font on google fonts:
![Captura de Pantalla 2022-09-27 a la(s) 8 55 12 a  m](https://user-images.githubusercontent.com/86022690/192546212-eff1a524-7faa-46bb-8517-3ab3be0562fd.png)

Hope this will work for you 😄